testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -20,6 +20,8 @@ metadata:
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
+  cleanup:
+    enabled: false
```

Comment:
That has no relation with the validator. 
That is just a change in the API.

Reply:
It is completely unrelated to the other content in this PR. Break this out into a separate PR.

Reply:
@estroz that is not possible at all. 
These files are updated by the make generate and are a consequence of the bump of the API.

Reply:
Ah I see. This is unfortunate since this field does not do anything, nor does the operator-sdk 
"support" this field. I'd like to see it stripped from the resulting manifest bytes before they are written to disk, in a separate PR.

Reply:
Done the bump in another PR: https://github.com/operator-framework/operator-sdk/pull/4750
That is fine. Tks.

Reply:
Done.